app/resources/forms.py

```diff
@@ -0,0 +1,32 @@
+from flask.ext.wtf import Form
+from wtforms.fields import (
+    StringField,
+    SubmitField
+)
+from wtforms.validators import InputRequired, Length
+
+
+class ResourceForm(Form):
+    name = StringField('Name', validators=[
```

Comment:
Should these fields also have `description`s (which would be displayed if one of the validators failed)?